client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/task-helper.ts

```diff
@@ -38,6 +38,7 @@ export function getEnhancedTasks(
 					taskData = {
 						title: translate( 'Choose a Plan' ),
 						keepActive: true,
+						hideChevron: true,
```

Comment:
I'm wondering if we can derive hideChevron from existing attributes? If I'm reading it right, seems like maybe hideChevron = isComplete && ! badgeText. If so, we could skip adding the extra task property and just compute the value dynamically in the ChecklistItem component.